forge-gui/res/cardsfolder/upcoming/solemn_doomguide.txt

```diff
@@ -0,0 +1,10 @@
+Name:Solemn Doomguide
+ManaCost:3 B B
+Types:Creature Tiefling Cleric
+PT:4/5
+K:Flying
+S:Mode$ Continuous | Affected$ Creature.Cleric+YouCtrl,Creature.Rogue+YouCtrl,Creature.Warrior+YouCtrl,Creature.Wizard+YouCtrl | AffectedZone$ Graveyard | AddKeyword$ Unearth:1 B | Description$ Each creature card in your graveyard that's a Cleric, Rogue, Warrior, and/or Wizard has unearth {1}{B}.
```

Comment:
Description$ should have the whole of the oracle here

Reply:
Updated SpellDescription$ to reflect whole of Oracle.